### PR TITLE
Fix RT

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by freearhey — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
All links are geo-blocked inside Europe. Please specify.

File: streams/ru.m3u [hunk 1]
```diff
@@ -112,16 +112,18 @@ https://rtmp.api.rt.com/hls/rtdru.m3u8
...
 #EXTINF:-1 tvg-id="RTDoc.ru" status="timeout",RT Documentary Russian (1080p) [Not 24/7]
 https://strm.yandex.ru/kal/rtd_hd/rtd_hd0.m3u8
-#EXTINF:-1 tvg-id="" status="error",RT en Español (1080p) [Not 24/7]
-https://rt-esp.gcdn.co/live/rtesp/playlist.m3u8
-#EXTINF:-1 tvg-id="RTFrance.ru" status="error",RT France (1080p)
-https://rt-fra.gcdn.co/live/rtfrance/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",RT en Español (1080p) [Not 24/7]
+https://rt-esp.rttv.com/live/rtesp/playlist.m3u8
+#EXTINF:-1 tvg-id="RTFrance.ru" status="online",RT France (1080p)
+https://rt-fra.rttv.com/live/rtfrance/playlist.m3u8
```
Comment (freearhey) [inline streams/ru.m3u:118]:
```suggestion
#EXTINF:-1 tvg-id="" status="online",RT en Español (1080p) [Geo-blocked]
https://rt-esp.rttv.com/live/rtesp/playlist.m3u8
#EXTINF:-1 tvg-id="RTFrance.ru" status="online",RT France (1080p) [Geo-blocked]
https://rt-fra.rttv.com/live/rtfrance/playlist.m3u8
```

File: streams/ru.m3u [hunk 2]
```diff
@@ -112,16 +112,18 @@ https://rtmp.api.rt.com/hls/rtdru.m3u8
...
-#EXTINF:-1 tvg-id="RTNews.ru" status="error",RT News (1080p) [Not 24/7]
-https://rt-glb.gcdn.co/live/rtnews/playlist.m3u8
-#EXTINF:-1 tvg-id="RTUK.ru" status="error",RT UK (1080p)
-https://rt-uk.gcdn.co/live/rtuk/playlist.m3u8
+#EXTINF:-1 tvg-id="RTNews.ru" status="online",RT News (1080p)
+https://live-hr6yv36f.rmbl.ws/slot-30/ngrp:srak-wx8r_all/playlist.m3u8
+#EXTINF:-1 tvg-id="RTNews.ru" status="online",RT News (1080p)
+https://rt-glb.rttv.com/live/rtnews/playlist.m3u8
+#EXTINF:-1 tvg-id="RTUK.ru" status="online",RT UK (1080p)
+https://rt-uk.rttv.com/live/rtuk/playlist.m3u8
```
Comment (freearhey) [inline streams/ru.m3u:126]:
```suggestion
#EXTINF:-1 tvg-id="RTNews.ru" status="online",RT News (1080p) [Geo-blocked]
https://live-hr6yv36f.rmbl.ws/slot-30/ngrp:srak-wx8r_all/playlist.m3u8
#EXTINF:-1 tvg-id="RTNews.ru" status="online",RT News (1080p) [Geo-blocked]
https://rt-glb.rttv.com/live/rtnews/playlist.m3u8
#EXTINF:-1 tvg-id="RTUK.ru" status="online",RT UK (1080p) [Geo-blocked]
https://rt-uk.rttv.com/live/rtuk/playlist.m3u8
```

File: streams/ru.m3u [hunk 3]
```diff
@@ -93,15 +93,15 @@ https://cdn-01.bonus-tv.ru/radostmoya_edge/index.m3u8
...
+#EXTINF:-1 tvg-id="RTArabic.ru" status="online",RT Arabic (1080p)
+https://rt-arb.rttv.com/live/rtarab/playlist.m3u8
 #EXTINF:-1 tvg-id="RTArabic.ru" status="error",RT Arabic (1080p) [Not 24/7]
 https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/channel/UCsP3Clx2qtH2mNZ6KolVoZQ/live
-#EXTINF:-1 tvg-id="RTDocumentary.ru" status="error",RT Documentary (1080p)
-https://rt-rtd.gcdn.co/live/rtdoc/playlist.m3u8
-#EXTINF:-1 tvg-id="RTDoc.ru" status="online" user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148",RT Documentary Russian (480p)
+#EXTINF:-1 tvg-id="RTDocumentary.ru" status="online",RT Documentary (1080p)
+https://rt-rtd.rttv.com/live/rtdoc/playlist.m3u8
+#EXTINF:-1 tvg-id="RTDoc.ru" status="timeout" user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148",RT Documentary Russian (480p)
```
Comment (freearhey) [inline streams/ru.m3u:104]:
```suggestion
#EXTINF:-1 tvg-id="RTDocumentary.ru" status="online",RT Documentary (1080p) [Geo-blocked]
https://rt-rtd.rttv.com/live/rtdoc/playlist.m3u8
#EXTINF:-1 tvg-id="RTDoc.ru" status="timeout" user-agent="Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148",RT Documentary Russian (480p) [Geo-blocked]
```

File: streams/ru.m3u [hunk 4]
```diff
@@ -93,15 +93,15 @@ https://cdn-01.bonus-tv.ru/radostmoya_edge/index.m3u8
...
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s96/index.m3u8
-#EXTINF:-1 tvg-id="RTAmerica.ru" status="error",RT America (1080p) [Not 24/7]
-https://rt-usa.gcdn.co/live/rtusa/playlist.m3u8
-#EXTINF:-1 tvg-id="RTArabic.ru" status="error",RT Arabic (1080p)
-https://rt-arb.gcdn.co/live/rtarab/playlist.m3u8
+#EXTINF:-1 tvg-id="RTAmerica.ru" status="online",RT America (1080p) [Not 24/7]
+https://rt-usa.rttv.com/live/rtusa/playlist.m3u8
+#EXTINF:-1 tvg-id="RTArabic.ru" status="online",RT Arabic (1080p)
+https://rt-arb.rttv.com/live/rtarab/playlist.m3u8
```
Comment (freearhey) [inline streams/ru.m3u:99]:
```suggestion
#EXTINF:-1 tvg-id="RTAmerica.ru" status="online",RT America (1080p) [Geo-blocked]
https://rt-usa.rttv.com/live/rtusa/playlist.m3u8
#EXTINF:-1 tvg-id="RTArabic.ru" status="online",RT Arabic (1080p) [Geo-blocked]
https://rt-arb.rttv.com/live/rtarab/playlist.m3u8
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Comment by BountyChocolate123456** [comment]
Thanks @freearhey & @LaneSh4d0w

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**